packages/icons/preview/import_statements.jsx

```diff
@@ -0,0 +1,16 @@
+import upperFirst from 'lodash/upperFirst';
+import camelCase from 'lodash/camelCase';
+
+const getIconName = (fileName) => fileName.substr(0, fileName.lastIndexOf('.'));
+
+export const importStatementForReactComponent = (size, fileName) => {
+  const name = getIconName(fileName);
+  const alias = upperFirst(camelCase(`${name}_icon`));
+  return `import { ReactComponent as ${alias} } from "@goodhood/icons/svg/${size}/${fileName}"`;
+}
+
+export const importStatementForImg = (size, fileName) => {
+  const name = getIconName(fileName);
+  const alias = camelCase(`${name}_icon`);
```

Comment:
i think in this case would make sense `whateverUrl` not `whatever_icon`

Reply:
Imo it would improve readability if the import alias has the word 'icon' in it.

WhateverIcon - React Component rendering the Icon
whateverIcon - URL to Icon

imo the information that it's a icon is more important than the data behind the alias. As soon as I see a icon with lowercase starting letter, I know it's a url.

In nebenan-frontend we already imported some svgs and we **never** applied the 'Url' suffix.

Reply:
![Screenshot 2020-05-15 at 18 29 19](https://user-images.githubusercontent.com/34499633/82073784-03434a80-96da-11ea-9db2-2e924e2f88ce.png)

IconUrl suffix is weird too